src/utils/Api.js

```diff
@@ -0,0 +1,123 @@
+class Api {
+  constructor({ baseUrl, headers = {} } = {}) {
+    this._baseUrl = baseUrl;
+    this._headers = headers;
+  }
+
+  // TODO create another method called get user info
+  getUserInfo() {
+    return fetch(`${this._baseUrl}/users/me`, {
+      headers: this._headers,
+    }).then((res) => {
+      if (res.ok) {
+        return res.json();
+      }
+      return Promise.reject(`Error: ${res.status}`);
+    });
+  }
+
+  getAppInfo() {
+    // TODO Call getuserinfo in this array
+    return Promise.all([
+      fetch(`${this._baseUrl}/cards`, {
+        headers: this._headers,
+      }).then((res) => res.json()),
```

Comment:
` .then(this._checkResponse)` will be used everywhere because you need to check every request if it's `ok` or not